GithubRepo/Scenes/Search/View/SearchViewController.swift

```diff
@@ -0,0 +1,177 @@
+//
+//  SearchViewController.swift
+//  GithubRepo
+//
+//  Created by Marcos Alves on 07/09/21.
+//
+
+import UIKit
+
+class SearchViewController: UIViewController {
+    
+    // MARK: - Constants
+    
+    private let kDebounceTime = 0.5
+    private let kMinStringToSearch = 3
+    private let kRepositoryTableViewCellIdentifier = String(describing: RepositoryTableViewCell.self)
```

Comment:
You can place this identifier inside the `RepositoryTableViewCell` as a static member